benchmarks/ps-experiment/llama-8b/ps_k8s_stack.yaml

```diff
@@ -0,0 +1,303 @@
+          - "--k8s-namespace"
+          - "default"
+          - "--k8s-label-selector"
+          - environment=test,release=test
+          - "--routing-logic"
+          - "session"
+          - "--session-key"
+          - "x-user-id"
+          - "--engine-stats-interval"
+          - "15"
+          - "--request-stats-window"
+          - "60"
+        resources:
+          requests:
+            cpu: "2"
+            memory: "8G"
+          limits:
+            cpu: "2"
+            memory: "8G"
+        ports:
+          - name: "router-cport"
+            containerPort: 8000
+
+        livenessProbe:
+          initialDelaySeconds: 30
+          periodSeconds: 5
+          failureThreshold: 3
+          httpGet:
+            path: /health
+            port: 8000
+---
+# Source: vllm-stack/templates/deployment-vllm-multi.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: "vllm-llama3-deployment-vllm"
+  namespace: default
+  labels:
+    model: llama3
+    environment: test
+    release: test
+spec:
+  replicas: 8
+  strategy:
+    rollingUpdate:
+      maxSurge: 100%
+      maxUnavailable: 0
+  selector:
+    matchLabels:
+      environment: test
+      release: test
+  progressDeadlineSeconds: 1200
+  template:
+    metadata:
+      labels:
+        model: llama3
+        environment: test
+        release: test
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: machine.cluster.vke.volcengine.com/gpu-name
+                    operator: In
+                    values:
+                      - Tesla-A100-80G
+      initContainers:
+        - command:
+            - aibrix_download
+            - --model-uri
+            - tos://aibrix-artifact-testing/models/llama-3.1-8b-instruct/
+            - --local-dir
+            - /models/
+          env:
+            - name: DOWNLOADER_NUM_THREADS
+              value: "16"
+            - name: DOWNLOADER_ALLOW_FILE_SUFFIX
+              value: json, safetensors
+            - name: TOS_ACCESS_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: TOS_ACCESS_KEY
+                  name: tos-credential
+            - name: TOS_SECRET_KEY
+              valueFrom:
+                secretKeyRef:
+                  key: TOS_SECRET_KEY
+                  name: tos-credential
+            - name: TOS_ENDPOINT
+              value: https://tos-s3-cn-beijing.ivolces.com
+            - name: TOS_REGION
+              value: cn-beijing
+          image: aibrix-container-registry-cn-beijing.cr.volces.com/aibrix/runtime:v0.2.1
+          name: init-model
+          volumeMounts:
+            - mountPath: /models
+              name: model-hostpath
+      containers:
+        - name: "vllm"
+          image: "aibrix-container-registry-cn-beijing.cr.volces.com/aibrix/vllm-openai:v0.7.0"
+          command:
```

Comment:
env variable for VLLM_RPC_TIMEOUT is missing.

Reply:
their generated one doesn't have it, but we can add it for fair comparison

Reply:
done